File: hw/gudino/u1/hw07ChickenCompleatVersionInPy&JS/poop.py
class Poop:
    def __init__(self, amount):
        self.amount = amount

    def get_amount(self):
        return self.amount

    def set_amount(self, amount):
        self.amount = amount

    def __str__(self):
        return f"Poop{{amount={self.amount}}}"

File: hw/gudino/u1/hw07ChickenCompleatVersionInPy&JS/poop.js
class Poop {
  constructor(amount) {
    this.amount = amount;
  }

  getAmount() {
    return this.amount;
  }

  setAmount(amount) {
    this.amount = amount;
  }

  toString() {
    return `Poop{amount=${this.amount}}`;
  }
}

export { Poop };
